featuretools/tests/entityset_tests/test_serialization.py

```diff
@@ -268,6 +268,13 @@ def test_serialize_url_csv(es):
         es.to_csv(URL, encoding='utf-8', engine='python')
 
 
+def test_serialize_data_description(es, tmpdir):
```

Comment:
this test name could be updated to provide more context about that it is testing extra files are not removed

Reply:
Is it okay now?

Reply:
Looks like there's an issue with py35 / py27 tests.

Reply:
Fixed :)